plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/buildlimit/BuildLimitsConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.programmerdan.minecraft.simpleadminhacks.configs.buildlimit;
+
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.framework.SimpleHackConfig;
+import com.programmerdan.minecraft.simpleadminhacks.framework.utilities.BuildLimit;
+import org.bukkit.configuration.ConfigurationSection;
+
+import java.util.List;
+
+public final class BuildLimitsConfig extends SimpleHackConfig {
+	private BuildLimit[] buildLimits;
+
+	public BuildLimitsConfig(SimpleAdminHacks plugin, ConfigurationSection base) {
+		super(plugin, base);
+	}
+	public BuildLimitsConfig(ConfigurationSection base) {
+		super(SimpleAdminHacks.instance(), base);
+	}
+
+	@Override
+	protected void wireup(ConfigurationSection config) {
+		if (this.shouldEnable()) {
+			List<?> rawList = config.getList("limits");
+			if (rawList != null && rawList.size() > 0) {
+				try {
+					this.buildLimits = rawList.toArray(new BuildLimit[rawList.size()]);
+				} catch(ArrayStoreException ase) {
+					ase.printStackTrace();
```

Comment:
This throwing is fine as it will disable the hack, which will be more visible to the administrator. Just printing the error like this will cause strange behaviour as the hack will still be enabled but not initialised properly.